src/models/CreatedBy.ts

```diff
@@ -1,3 +1,3 @@
 export interface CreatedBy {
-    _id: string;
+    id: string;
```

Comment:
does this mean we can remove the  `@JsonAlias({"_id"})` from the api?

Reply:
yep :)